prometheus/testutil/testutil.go

```diff
@@ -40,13 +40,12 @@ package testutil
 import (
 	"bytes"
 	"fmt"
-	"io"
-	"reflect"
-
 	"github.com/davecgh/go-spew/spew"
-	"github.com/prometheus/common/expfmt"
-
 	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
```

Comment:
Nit: I believe the ordering and grouping imports has a reasoning (first group is stdlib, second group is 3rd parties and third group is "local imports") and shoudn't be changed without a solid reason.

[goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports#pkg-overview) is a tool, that might automates this.